compiler/resolution/functionResolution.cpp

```diff
@@ -8528,6 +8532,8 @@ static void removeUnusedFunctions() {
 
           clearDefaultInitFns(fn);
 
+          cleanupIteratorGroup(fn);
```

Comment:
I think it's simpler to simply set some iteratorInfo->iteratorGroup elements to NULL in ~FnSymbol

Reply:
Since you now call this in ~FnSymbol is it also necessary to call here?